db/schema.rb

```diff
@@ -54,12 +54,12 @@
     t.text     "comments"
     t.boolean  "adult"
     t.boolean  "appropriate_adult",         default: false, null: false
+    t.string   "state"
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "state"
     t.string   "dscc_number"
-    t.text     "feedback"
     t.datetime "interview_start_time"
+    t.text     "feedback"
```

Comment:
I think these changes are fine now but if we were in production you'd need to be careful about not getting your schema columns out of order with the production database.